src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -31,35 +43,41 @@ public String todayDate(DateTimePart datePart) {
      * Return Optional of a date built from these elements.
      */
     public Optional<LocalDate> getDate(Integer[] dateParams) {
-        return Optional.empty();
+        LocalDate date;
+        try {
+            date = LocalDate.of(dateParams[0], dateParams[1], dateParams[2]);
+        } catch (Exception e) {
```

Comment:
you know that it isn't correct to catch Exception here and you work with ArrayIndexOutOfBoundsExceptin

Reply:
done